components/user_settings/notifications/email_notification_setting/email_notification_setting.tsx

```diff
@@ -246,7 +247,7 @@ export default class EmailNotificationSetting extends React.PureComponent<Props,
                             />
                         </div>,
                     ]}
-                    server_error={this.props.serverError}
+                    serverError={this.props.serverError}
```

Comment:
i am not sure if we just use the `server_error` in props of component instead. too many places this was changed. cc @AshishDhama

Reply:
@M-ZubairAhmed I think `serverError` matches with our naming convention so this change should be acceptable even if it is getting changed at multiple places.